commit 49519ba3f2d4874bde93a4504d97dbfcbe673619 (refs/changes/39/2139/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-06-23T11:14:04-07:00 (2 years, 4 months ago)
    
    MANTA-3322 want 'severity' column for 'manta-adm alarm list'
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/lib/adm.js b/lib/adm.js
index 5876623..1f02814 100644
--- a/lib/adm.js
+++ b/lib/adm.js
@@ -132,7 +132,7 @@ var maAlarmColumns = {
     },
     'dateopened': {
 	'label': 'DATE_OPENED',
-	'width': 10
+	'width': 11
     },
     'timeopened': {
 	'label': 'TIME_OPENED',
@@ -140,7 +140,7 @@ var maAlarmColumns = {
     },
     'dateclosed': {
 	'label': 'DATE_CLOSED',
-	'width': 10
+	'width': 11
     },
     'timeclosed': {
 	'label': 'TIME_CLOSED',
@@ -156,7 +156,7 @@ var maAlarmColumns = {
     },
     'nevents': {
 	'label': 'NEVENTS',
-	'width': 5,
+	'width': 7,
 	'align': 'right'
     },
     'nflts': {
@@ -168,6 +168,10 @@ var maAlarmColumns = {
 	'label': 'NFY',
 	'width': 3
     },
+    'severity': {
+	'label': 'SEVERITY',
+	'width': 8
+    },
     'summary': {
 	'label': 'SUMMARY',
 	'width': 30
@@ -1229,6 +1233,7 @@ maAdm.prototype.alarmsList = function (args)
 		    'NFLTS': alarm.a_faults.length,
 		    'NEVENTS': alarm.a_nevents,
 		    'NFY': alarm.a_suppressed ? 'no' : 'yes',
+		    'SEVERITY': details.ka ? details.ka.ka_severity : '-',
 		    'SUMMARY': details.summary
 		});
 	});
